osversion/platform_compat_windows.go

```diff
@@ -0,0 +1,70 @@
+package osversion
+
+import (
+	"strconv"
+	"strings"
+)
+
+// List of stable ABI compliant
+// ltsc releases
+// Note: List must be sorted
+var compatLtscReleases = []uint{
+	V21H2Server,
+}
+
+// Checks for stable ABI compliant OS versions.
+// Every release after WS 2022 will support the previous
+// ltsc container image. Stable ABI is in preview mode
+// for windows 11 client.
+// Refer: https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-10#windows-server-host-os-compatibility
+func CanRun(hostOSVersion, guestOSVersion string) bool {
```

Comment:
I don't think `CanRun` is a great name for this (I know I used it in my example, but we should find a better name for the actual code :)). Maybe `ContainerIsABICompatible`?

Reply:
I thought it was good as this function takes care of both stableABI and non-stable ABI compliant versions. I'll change it to AreVersionsCompatible()

Reply:
Not sure ContainerIsABICompatible() would be apt here as this function is doing the matching for both stable ABI compliant and prior versions

Reply:
I think your current name is okay, but even for older releases this function is still checking ABI compatibility, so the name would still make sense. It's just that on older versions the host/guest are only ABI compatible if they are on the same build number.